how-to/install/enable-network-role.rst

```diff
@@ -0,0 +1,98 @@
+Enable Network Role for Canonical Openstack
```

Comment:
This page should not exist in install. The information should be moved to different pages as and when we see necessary.